src/main/java/be/seeseemelk/mockbukkit/block/BlockMock.java

```diff
@@ -457,6 +457,20 @@ public boolean breakNaturally(@NotNull ItemStack tool, boolean triggerEffect)
 
 	}
 
+	@Override
+	public void tick()
+	{
+		// TODO Auto-generated method stub
+		throw new UnimplementedOperationException();
+	}
+
+	@Override
+	public void randomTick()
+	{
+		// TODO Auto-generated method stub
+		throw new UnimplementedOperationException();
+	}
+
```

Comment:
Those Methods seem to be out of scope for this PR, feel free to open a separate PR to fix those